src/main/java/com/mini/test/Service/MapService.java

```diff
@@ -0,0 +1,11 @@
+package com.mini.test.Service;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+public interface MapService {
+    public void moveBackup(int xmin, int xmax, int ymin, int ymax);
+    public List<Path> getFilesRange(String basePath, int xmin, int xmax, int ymin, int ymax) throws IOException;
+    public boolean isRange(Path file, int xmin, int xmax, int ymin, int ymax);
```

Comment:
xmin, xmax, ymin, ymax 의 int 를 가지고 있는 class dto 를 하나 만들어주면 더 좋을 것 같네
```suggestion
    public void moveBackup(BoundingBoxDto box);
    public List<Path> getFilesRange(String basePath, BoundingBoxDto box) throws IOException;
    public boolean isRange(Path file, BoundingBoxDto box);
```

Reply:
넵